docs/migrate-module.md

```diff
@@ -0,0 +1,94 @@
+# Update an older module
+
+Module repos created before July 2022 don't include the logic that is needed to pass the continuous integration (CI) tests and to publish the modules to the IBM Cloud catalog. Follow these steps to update an existing repo in the terraform-ibm-modules GitHub organization.
```

Comment:
This applies to any existing module, not just the existing modules repos in terraform-ibm-modules GitHub organization. For instance the SLZ modules come from internal GHE repos